detox/test/e2e/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "setupTestFrameworkScriptFile" : "./helpers/init.js",
+  "bail": true,
+  "verbose": true,
+  "forceExit": true
```

Comment:
We should validate Detox doesn't leave anything open, and that everything is being closed correctly.
Please remove `forceExit`